<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerWidget</class>
 <widget class="QWidget" name="ControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="OpenGLWidget" name="m_OpenGLWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="m_LeftRightLabel">
           <property name="text">
            <string>Left Right</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="m_UpDownLabel">
           <property name="text">
            <string>Up Down</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="m_AnchorXSlider">
           <property name="minimum">
            <number>-640</number>
           </property>
           <property name="maximum">
            <number>640</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="m_ZoomXLabel_2">
           <property name="text">
            <string>AnchorX</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="m_RotateLabel">
           <property name="text">
            <string>Rotation</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="m_ResetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="m_UpDownSlider">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSlider" name="m_ZoomYSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="m_RotationSlider">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="m_ZoomYLabel">
           <property name="text">
            <string>ZoomY</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSlider" name="m_ZoomXSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="m_LeftRightSlider">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="m_AnchorYSlider">
           <property name="minimum">
            <number>-540</number>
           </property>
           <property name="maximum">
            <number>540</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="m_ZoomYLabel_2">
           <property name="text">
            <string>Anchor Y</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="m_ZoomXLabel">
           <property name="text">
            <string>ZoomX</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="m_FrontBackSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>106</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>OpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
